v0/destinations/customerio/transform.js

```diff
@@ -36,6 +36,16 @@ const deviceRelatedEventNames = [
   "Application Opened",
   "Application Uninstalled"
 ];
+
+const isdeviceRelatedEventName = (eventName, destination) => {
+  return (
+    deviceRelatedEventNames.includes(eventName) ||
+    (destination.Config &&
+      destination.Config.deviceTokenEventName &&
+      destination.Config.deviceTokenEventName === eventName)
```

Comment:
I think this is different from what was decided.
Basically my perspective of the decision made doesn't involve the destination's configuration(I mean UI), I was thinking more from a perspective that an event(Device Registration) is triggered from the user's side after we get the device token.

Let me know if there is some mis-understanding here